### ISSUE TITLE
Touch Not Respecting "disabled-on-external-mouse" mode

### ISSUE THREAD

**Issue by ayu2805** [JoseExposito/touchegg#655]
### Description of the bug

I am having a laptop system configured with `gsettings set org.gnome.desktop.peripherals.touchpad send-events disabled-on-external-mouse` but in X11(obviously) touchegg is forcing the touchpad to recieve multiple touch even though basic functionality is gone(i.e. touch to move cursor, touchpad buttom click and tap to click).

### Expected behaviour

Touchpad shouldn't receive any kind of touch with this configuration:
`gsettings set org.gnome.desktop.peripherals.touchpad send-events disabled-on-external-mouse`

### Actual behaviour

Touchpad is receiving 3 finger swipe gestures

### My environment

 * Version of Touchégg: 2.0.17-1
 * Operating System: Arch Linux
 * Desktop Environment: Gnome

**Comment by JoseExposito** [comment]
> touchegg is forcing the touchpad to recieve multiple touch even though basic functionality is gone(i.e. touch to move cursor, touchpad buttom click and tap to click).

Are you able to move the pointer while you are typing only when Touchégg is running?
That's unexpected. It is true that Touchégg doesn't follow the disable while typing preferences for gestures (i.e, swipe and pinch) but it shouldn't affect cursor movement.

Try to kill the Touchégg process (`killall touchegg`, **no** sudo!) and see how your touchpad behaves.
If disable while typing is not working, open a bug in libinput and I'll help you there.

**Comment by ayu2805** [comment]
> Are you able to move the pointer while you are typing only when Touchégg is running?

I think you misunderstood. Obviously I cannot move the pointer while typing, but my issue was that the touchpad is receiving only multiple touch gestures even when I have set `gsettings set org.gnome.desktop.peripherals.touchpad send-events disabled-on-external-mouse`
This option is intended to disable the touchpad when an external mouse is attached. And this is indeed functional in wayland(where touchegg doesn't work) but gets buggy with touchegg(X11) and starts receiving only multiple touch

**Comment by ayu2805** [comment]
Was it reproducible? Or something is wrong only from my side?

**Comment by JoseExposito** [comment]
Yes, it is reproducible. This a feature not implemented in Touchégg.

**Comment by ayu2805** [comment]
Okay. Thank you for your time and assistance. I appreciate your help and eagerly anticipate the forthcoming update.
